### PR TITLE
Issue 4169 - UI - port charts to PF4

### PR DESCRIPTION
**Pull request by mreynolds389**
Description:  

Ported the charts under the monitor tab to use PF4 sparkline charts and provide realtime stats on the caches.

Relates: https://github.com/389ds/389-ds-base/issues/4169

### CODE REVIEW

File: src/cockpit/389-console/src/lib/tools.jsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export function get_date_string(timestamp) {
         parseInt(minute),
         parseInt(sec)
     );
-    return date.toLocaleString();
+    return date.toLocaleString('en-ZA');
```
Comment (droideck) [inline src/cockpit/389-console/src/lib/tools.jsx:78]: Hm, is there any particular reason for 'en-ZA' locale? (ZA part confuses me)

**Review by droideck — APPROVED** [review]
Besides this minor question, looks really good to me!

Charts are awesome!

File: src/cockpit/389-console/src/lib/tools.jsx (hunk 1 above)
Comment (mreynolds389) [reply src/cockpit/389-console/src/lib/tools.jsx:78]: Good question.  So in the monitor the server start date uses MM/DD/YYYY, but I want it to use a more universal format.  Locale "en-ZA" uses YYYY/MM/DD

I'm open to other suggestions as well.

Comment (droideck) [reply src/cockpit/389-console/src/lib/tools.jsx:78]: Agree, it's a nice format. You have my full ack :)

**Pull request merged**